scipy/stats/tests/test_distributions.py

```diff
@@ -1195,6 +1195,17 @@ def test_expect(self):
 
         assert_allclose(rv.expect(), np.sum(rv.xk * rv.pk), atol=1e-14)
 
+    def test_multidimension(self):
+        xk = np.array([[1,2,3,4],
```

Comment:
to have a shorter test, use np.arange(12).reshape((3, 4)); (don't forget space after ,)

Reply:
Thanks. I will use `np.arange()` for this test case.